bundle/manifests/liberty.websphere.ibm.com_webspherelibertyapplications.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.9.2
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Please delete `/bin/controller-gen` and `/bin/kustomize` and re-run the `make bundle generate manifests bundle` command to generate the CRD with the right (updated version)